addons/im_livechat/views/rating_views.xml

```diff
@@ -26,7 +26,6 @@
         <field name="name">Ratings for livechat channel</field>
         <field name="res_model">rating.rating</field>
         <field name="view_mode">kanban,tree,graph,pivot,form</field>
-        <field name="domain">[('parent_res_model','=','im_livechat.channel'), ('consumed','=',True)]</field>
```

Comment:
Why did you need to remove this?
Doesn't this make this default action (when not overridden) incorrect?

Reply:
Note: it would be nice is the stat button was using the new `action_view_channel_ratings` too. Currently it's strange to have a different action open when using the kanban icon or the stat button from form view.